src/monexamine.cpp

```diff
@@ -101,6 +101,21 @@ void remove_saddle_from( monster &z )
     z.tack_item.reset();
 }
 
+void bandage_animal( monster &z )
+{
+    if( z.has_effect( effect_bleed ) ) {
```

Comment:
`effect_bleed` also needs to be declared at the top of the file.

Reply:
Thanks! I haven't had a chance to get back to this one after opening it.